src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/GenerateDefendantResponseReceiptTest.java

```diff
@@ -54,6 +62,8 @@ public void shouldReturnServerErrorWhenPdfGenerationFails() throws Exception {
 
     private ResultActions makeRequest(String externalId) throws Exception {
         return webClient
-            .perform(get("/documents/defendantResponseReceipt/" + externalId));
+            .perform(get("/documents/defendantResponseReceipt/" + externalId)
+                .header(HttpHeaders.AUTHORIZATION, AUTHORISATION_TOKEN)
```

Comment:
What do you think about moving `makeRequest` to `BaseIntegrationTest` and changing it to `makeAuthorisedRequest(String url)`? This way we wouldn't have to repeat this bit of code and the constant in a number of places.